index.js

```diff
@@ -379,8 +379,10 @@ const createFocusTrap = function (elements, userOptions) {
   // so that it precedes the focus event.
   const checkPointerDown = function (e) {
     const target = getActualTarget(e);
+    const composedPath = typeof e.composedPath === 'function' && e.composedPath();
+    const clickedInsideContainer = composedPath && state.containers.some(container => composedPath.includes(container));
 
-    if (findContainerIndex(target) >= 0) {
+    if (findContainerIndex(target) >= 0 || clickedInsideContainer) {
```

Comment:
I'm wondering if we need both of these checks?

Reply:
Actually, the `composedPath()` check should be _inside_ `findContainerIndex()` so that the index is returned if the `target` is indeed inside one of the containers. That way, we cover other places where `findContainerIndex()` is called for the same purpose.

Reply:
Done ✅